general/management/commands/gm_worker_async_tasks.py

```diff
@@ -95,7 +96,11 @@ def task_email_random_sound(self, gearman_worker, gearman_job):
         random_sound_id = gearman_job.data
         random_sound = Sound.objects.get(id=random_sound_id)
 
-        send_mail_template(\
+        disabled = UserEmailSetting.objects.filter(user=random_sound.user,
+                                email_type__name="random_sound").count()
```

Comment:
I know this is not easy, but I think we need a helper for this... Using raw strings for definite things always seems risky... You could have a utility function that throws an exception for example:

UserEmailSetting.objects.filter(user=random_sound.user, email_type=get_email_type("random_sound")).count()

so that at least the string is "checked at runtime"...